game.rb

```diff
@@ -47,4 +57,19 @@ def draw_background
 
 end
 
-Game.run
+# game = Game.new
+# Thread.new {
+#   game.show
+# }
+# game.draw
+# game.update
+# game.draw
+# puts "Game field: #{game.board.field}"
+
+# game.board.reveal_at({x: 3, y: 0})
+
+# Game.new.show
+
+# while 1
+# end
```

Comment:
And FYI: `Game.run` is Hasu's version, which ends up calling reset automatically.  I'm not sure if it works properly if you do the vanilla Gosu `Game.new.show`.